client/strings/en.conf

```diff
@@ -341,13 +343,12 @@
 !system 1355 Hide Chain Buttons
 !system 1360 Previous
 !system 1370 Level↑
-!system 1371 Attack ↑
-!system 1372 Defense ↑
-!system 1373 Name ↓
+!system 1371 ATK↑
+!system 1372 DEF↑
+!system 1373 Name↓
 !system 1390 Waiting...
 !system 1391 Waiting....
 !system 1392 Waiting.....
-!system 1393 Archetype:
 !system 1400 Unable to connect to the host.
 !system 1401 Connection has been lost.
 !system 1402 An error occurred during the transmission.
```

Comment:
Ah ok , i assume its needed because Salvations source code is different than the official ? because its not in the official strings

Also I would wait on these

    !system 522 请选择表侧攻击表示的怪兽 -- Translated as "Please select the side of the monster that attack"
    !system 523 请选择表侧守备表示的怪兽 -- Translated as "Please select the table side garrison that monster"
    !system 524 Select attack monster
    !system 525 Select defense monster
    !system 566 请选择要发动效果的卡 -- Translated as "Please select the card to be activated"
    !system 568 请选择要处理效果的卡 -- Translated as "Please select the card to process the effect"

since im still unsure about them